pkgs/data/fonts/vollkorn/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchzip }:
+let
+  version = "4.105";
+in fetchzip rec {
+  name = "vollkorn-${version}";
+
+  url = "http://vollkorn-typeface.com/download/vollkorn-4-105.zip";
+
+  downloadToTemp = true;
+  recursiveHash = true;
+  postFetch = ''
+    mkdir -p $out/share/{doc,fonts}
+    unzip -j $downloadedFile \*.ttf   -d $out/share/fonts/truetype
+    unzip -j $downloadedFile \*.otf   -d $out/share/fonts/opentype
+    unzip -j $downloadedFile \*.woff  -d $out/share/fonts/woff
+    unzip -j $downloadedFile \*.woff2 -d $out/share/fonts/woff2
+    unzip -j $downloadedFile \*.eot   -d $out/share/fonts/EOT
+    unzip -j $downloadedFile \*.txt   -d "$out/share/doc/${name}"
+  '';
+
+  sha256 = "0pvy72v8wsnghww67vy8i1wyk4nikyh8pylk7gjp21ar754sih3k";
+
+  meta = with lib; {
```

Comment:
A package needs a `maintainer` :)

Please add yourself (as you created the package), and also add yourself to maintainers/maintainer-list.nix in a separate commit.